src/stories/shared/select/MemberSelect.stories.tsx

```
@@ -0,0 +1,138 @@
+    imageUrl: 'https://avatars.githubusercontent.com/u/2345678?v=4',
+  },
+  {
+    id: 6,
+    nickname: '망곰곰곰맨',
+    imageUrl: 'https://avatars.githubusercontent.com/u/2345678?v=4',
+  },
+]
+
+export const Default: Story = {
+  render: () => {
+    const [selectedMembers, setSelectedMembers] = useState<MemberInfo[]>([])
```

ESLint

[error] React Hook "useState" is called in function "render" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)